web_src/css/repo/issue-list.css

```diff
@@ -1,15 +1,61 @@
-.issue-list-toolbar {
-  display: flex;
-  flex-wrap: wrap;
-  align-items: flex-start;
-  gap: 1rem;
-  margin-top: 1rem;
+@media only screen and (min-width: 1024px) and (max-width: 1266px) {
+  #issue-filters > .column.six.wide.column {
+    max-width: fit-content;
+  }
+
+  #issue-filters > .issue-list-toolbar-right > .stackable.menu {
+    margin-left: calc(33% - 7vw);
+  }
+}
+
+@media only screen and (min-width: 1266px) {
```

Comment:
This 1266px does not match any of our other media queries, I think the right one is 1200px.

Reply:
Changed to 1200px.